js/model.js

```diff
@@ -0,0 +1,91 @@
+import modeldata from './modelData.js';
+
+document.querySelector('.Grid-boxes').innerHTML = modeldata
+  .map(
+    (data, index) => `
+    <article class='Box'>
+    <img class='box-image' src='${data.snapshot}' alt='protfolio image tonic'>
+    <div class='content'>
+    <h2 class='content-title'>${data.projectname}</h2>
+    <div class='text-title'>
+        <a href='#' class='client'>${data.frame[0]}</a>
+        <img class='dot' src='${data.dot}' alt='dot'>
+        <a href='#' class='client-skills'>${data.frame[1]}</a>
+        <img class='dot' src='${data.dot}' alt='dot'>
+        <a href='#' class='period'>${data.frame[2]}</a>
+    </div>
+    <p class='content-text'>${data.para}</p>
+    <ul class='tags'>
+        <li class='style'>${data.technologies[0]}</li>
+        <li class='style'>${data.technologies[1]}</li>
+        <li class='style'>${data.technologies[2]}</li>
+    </ul>
+    <a href='#' class='link' aria-label='Enter here to see this project.'
```

Comment:
- [ ] A quick improvement here for better user experience would be to use a `<button>` element instead, using an anchor tag isn't quite semantically correct, since you're not navigating through your page or getting redirected elsewhere. 🤔 What's happening at the moment is that each time a user clicks on "See Project" he gets redirected to `#` as stated in your `href` attribute, ending up at the top of the page. Please consider fixing this behavior! 🤓 🤖

Reply:
thanks i did it